### PR TITLE
kibana, elasticsearch, logstash 6.4.1

### PR DESCRIPTION
**Pull request by chrmoritz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

~~This PR is blocked on upgrading other elastic formulae to a version compatible with `elasticsearch` v6.4.0. Refs: https://github.com/Homebrew/homebrew-core/pull/28995#issuecomment-419636367~~

<details><summary>ToDo list:</summary>

* [x] Make sure that we install the [correct license file](https://github.com/elastic/kibana/blob/v6.4.0/licenses/APACHE-LICENSE-2.0.txt).
* [x] Is there a reason we use a git tag for the bundled `node` instead of the source tarball? 
  * [#20638 (comment)](https://github.com/Homebrew/homebrew-core/pull/20638/files#r151140047) is resolved now => switching back to source tarball
* [x] Figure out why `yarn` persists on using `node` (`@10`) over `node@8` 
  * how does `yarn` even find `node@10` with this `echo $PATH`: `/usr/local/Cellar/kibana/6.4.0/libexec/node/bin:/usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/yarn/bin:/usr/bin:/bin:/usr/sbin:/sbin`?
  * even if you call `yarn.js` with the `node@8` executable it will still spawn `node-gyp` with `node@10` causing build issues: 
    ```ruby
    system prefix/"libexec/node/bin/node", Formula["yarn"].libexec/"bin/yarn.js", "kbn", "bootstrap"
    ```
</details>

### CODE REVIEW

File: Formula/kibana.rb
Comment (chrmoritz) [inline Formula/kibana.rb]: Is there a way to prevent homebrew from installing the top level `LICENSE.txt` and overwriting the correct license, if I install the license here directly with: `prefix.install "licenses/APACHE-LICENSE-2.0.txt" => "LICENSE.txt"`?

**Comment by chrmoritz** [comment]
This is still blocked on updating the other elastic formulae to v6.4.0, see https://github.com/Homebrew/homebrew-core/pull/28995#issuecomment-419969991.

**Comment by chrmoritz** [comment]
I've updated all 3 to the new version 6.4.1.

Can someone please take another look at this PR please. With the upcoming mac OS mojave release and the inability to bottle `kibana` on it (#31741), it might be worth merging this at it is. According to [elastics compatibility table](https://www.elastic.co/support/matrix#matrix_compatibility) we would still be compatible to the other `beats` formulae, which are still stuck on 6.2.4 for now. Also looking at the download statistics (install on request last month), `elasticsearch` (5600+) and `kibana` (1900+) are way more important/popular than the other `beats` formulae with less then 50 downloads a month (except `filebeat` with 250).

cc @DomT4, @fxcoudert, @jasontedor

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@chrmoritz thanks, I agree with your reasoning, although I'll leave some time for other maintainers to step in if they have an opinion on this

**Comment by DomT4** [comment]
> cc @DomT4

Just FYI, I've left and do not intend to keep using _(or contributing to)_ the project long-term, so I'm a useless ping 😉. Everything I was the point of contact for, you'll want to nudge FX on going forwards. Thanks.

**Comment by fxcoudert** [comment]
Many thanks @chrmoritz !

**Pull request closed without merging**
